cts/cts-cli.in

```diff
@@ -1205,15 +1205,15 @@ class ErrorCodeRegressionTest(RegressionTest):
         #
         # Don't test positive (system) error codes, which may vary by OS
         standard_tests = [
-            make_test_group("Get unknown Pacemaker return code", "crm_error -r {fmt} -- -10000"),
+            make_test_group("Get unknown Pacemaker return code", "crm_error -r {fmt} 10000"),
             make_test_group("Get unknown Pacemaker return code (with name)",
-                            "crm_error -n -r {fmt} -- -10000"),
+                            "crm_error -n -r {fmt} 10000"),
             # Negative return codes require parsing out the "--" explicitly, so we need
             # to test them as a separate case
-            make_test_group("Get negative Pacemaker return code", "crm_error -r {fmt} -- -1005"),
+            Test("Get negative Pacemaker return code", "crm_error -r -- -1005"),
```

Comment:
On this and the next one, `{fmt}` got removed instead of `--`. But see my other comment.